curvefs/src/client/s3/disk_cache_write.cpp

```diff
@@ -100,13 +102,18 @@ int DiskCacheWrite::UploadFile(const std::string name) {
         posixWrapper_->close(fd);
         return -1;
     }
-    ret = client_->Upload(name, buffer, fileSize);
-    if (ret < 0) {
-        LOG(ERROR) << "upload object fail. object: " << name;
-        posixWrapper_->free(buffer);
-        posixWrapper_->close(fd);
-        return -1;
-    }
+    PutObjectAsyncCallBack cb =
```

Comment:
This definition in many places，make it public

Reply:
The callback content is not the same